### PR TITLE
Update README and descriptions. Add DB cluster parameter group

### PR DESCRIPTION
**Pull request by s2504s**
## what
* Updated `README`
* Added variable descriptions
* Added DB cluster parameter group

## why
* Usage examples, Inputs and Outputs description
* Variable descriptions to follow best practices
* To be able to provide DB cluster parameters

## references
* https://www.terraform.io/docs/providers/aws/r/rds_cluster_parameter_group.html

### CODE REVIEW

**Comment by s2504s** [comment]
Still `WIP`

**Comment by lortega** [comment]
@s2504s maybe you should also explain the minimum possible variables needed to use the module. Sometimes, like me yesterday, you only want to give the module a quick try...

File: README.md [hunk 1]
```diff
@@ -6,3 +6,25 @@ Terraform module to provision an [`RDS Aurora`](https://aws.amazon.com/rds/auror
...
 
 Apache 2 License. See [`LICENSE`](LICENSE) for full details.
+
+
+## Usage
+
+Include this repository as a module in your existing terraform code:
+
+```terraform
+module "rds_cluster_aurora_acig" {
```
Comment (aknysh) [inline README.md]: should not use `asig` in the generic module, it's a customer specific name.
`module "rds_cluster_aurora"` should be enough.

File: README.md [hunk 2]
```diff
@@ -6,3 +6,25 @@ Terraform module to provision an [`RDS Aurora`](https://aws.amazon.com/rds/auror
...
+  source              = "git::https://github.com/cloudposse/terraform-aws-rds-cluster.git?ref=tags/0.2.1"
```
Comment (aknysh) [inline README.md]: should use the `master` branch in `README` examples.
git::https://github.com/cloudposse/terraform-aws-rds-cluster.git?ref=master

**Pull request merged**